General translation update

Rekoiru opened the pull request:
- General translations update. (Sadly, hashed has to be used again for some strings...)
- Reduced the font size by 12% with tweaked tracking. (Supports both windows & android)
- Tweaked config.json to better support the font.
- Banners up to date.
- Updated and added new atlases.
- Added sprites related to shop and races.
- Added some tl to banners on menus.
- Legends Scenario is fully translated.
- Island Scenario is fully translated.
- Removed "at a random point" from skill description. (it only made skill description larger for no reason, we know its rng already)
- Some chara/cards names, skill descriptions/names will be updated at a later date.

Commit:
General translation update

- General translations update. (Sadly, hashed has to be used again for some strings...)
- Reduced the font size by 12% with tweaked tracking. (Supports both windows & android)
- Tweaked config.json to better support the font.
- Banners up to date.
- Updated and added new atlases.
- Added sprites related to shop and races.
- Added some tl to banners on menus.
- Legends Scenario is fully translated.
- Island Scenario is fully translated.
- Removed "at a random point" from skill description. (it only made skill description larger for no reason, we know its rng already)
- Some chara/cards names, skill descriptions/names will be updated at a later date.

Rekoiru commented:
I've updated a ton of things on dict&mdb, some other on hashed. I got more to upload so I'm considering this part 1 of the upload till I update some files. The rest of the files can make the update go over 600mb~ so, that's why I went with this approach.
Review for changes are optional, it can contain some typos or grammar errors due to the large amount of text I've edited.

Commit:
Font files

Oopsies...

Commit:
Minor fixes

Commit:
New & Reworked sprites

- Reworked many sprites that were already done.
- Added new atlases and sprites.
- Refined some translations.

Commit:
Race sprites & metafiles

- Added race sprites translated.
- Added more metafiles.

Rekoiru commented:
Seems ready by now, anything missing is on the works yet. It can be pushed, the overall size of the folder may be around 600mb~ as I said so we may need to warn people about this update.

Commit:
Minor adjustments

- Minor translation adjustments on mdb & dict.
- Minor change in stadium atlas.
- Added a few untranslated strings.

Commit:
Banner update & stroll feature

- Added stroll feature tl.
- Fixes here and there (atlas).
- Updated to match the latest update.

LeadRDRK commented:
sorry for the late reply, not exactly free rn, will review your changes as soon as i have the chance

Rekoiru commented:
No problem, there's no need to rush things, in fact probably there will be some changes to be made or new tld things for the new landscape update or whatever.

Commit:
Minor adjustments

- Minor adjustments on dict & sprites
- Added more mdb entries
- Added landscape mode tl

Commit:
Banner update & older scenario sprites

- Adjusted and added more tl'd entries on dict, mdb and chara data.
- Added sprites and texts for older scenarios.
- Added some misc sprites & meta files.

Commit:
Atlas update

- Updated atlas to match latest update.

Commit:
Metafile update

- Updated metafiles to match latest update.
- Minor adjustments to some sprites & dict/mdb.

Commit:
Banner update 25-09-29

- Updated to the latest banner.
- Skills and descriptions will be changed later.
- Minor adjustments to dict&mdb.

Commit:
Minor adjustments

- Added a few sprites.
- Adjusted some atlases.
- Minor text adjustments.

Commit:
Banner update 25-10-10

- Updated to the latest banner.
- Minor fixes and adjustments to terms in dict/mdb post Support Points removal.
- Skills descriptions adjusted.
- Updated sprites & meta files to latest update.
